app/utils.py

```diff
@@ -1,11 +1,11 @@
+# TODO - This file contains multiple functions that contain import statements?  Is this to avoid a circular
+# dependency?  If not, import only at the beginning of the file.
```

Comment:
Please create a ticket for new TODOs and then reference the ticket number in this TODO comment.

Reply:
done